src/mover.py

```diff
@@ -6177,7 +6180,10 @@ def connect_client(self):
 
             #self.listen_socket.listen(1)
             if (not self.method) or (self.method and self.method != 'read_next'):
+                Trace.trace(10, 'connect_client: listening')
```

Comment:
Are these traces needed. They seem like debug leftovers.

Reply:
They are to investigate problems with client connection